### DISCUSSION TITLE
Elliptical Sources

### DISCUSSION THREAD

**Discussion by jenniferyee** [rpoleski/MulensModel#146] [Ideas]
This paper by Heyrovsky & Loeb: https://ui.adsabs.harvard.edu/abs/1997ApJ...490...38H/abstract

was brought to my attention. It provides analytic results for calculating FSPL magnification for an elliptical source. This is another magnification method that could be implemented in the future.

**Comment by rpoleski** [comment]
I think Nick Rattenbury published a paper with an estimate of source ellipticity. It's the only paper about that I remember.
